flocker/control/_protocol.py

```diff
@@ -137,12 +163,32 @@ def connectionLost(self, reason):
         self.control_amp_service.disconnected(self)
 
 
+DEPLOYMENT_CONFIG = Field(u"configuration", repr, u"The cluster configuration")
+CLUSTER_STATE = Field(u"state", repr, u"The cluster state.")
+
+LOG_SEND_CLUSTER_STATE = ActionType(
+    "flocker:controlservice:send_cluster_state",
```

Comment:
Again I find myself wondering how these strings are constructed and how anyone will make sense of them.

:question: